models/public/breast_ultrasound_simulation/README.md

```diff
@@ -0,0 +1,73 @@
+We further derive the relationship between the frequency of the transducer and the resolution of the ultrasound image to be able to simulate and form images at frequencies other than at which the model is trained. This is achieved by interpolation between dilated kernels in a multi-linear separable CNN used to control the resolution of the generated image allowing for zero-shot adaptation. Using the multi-linear separable design,, we can simulate entire 3D volumes by training the model only on 2D ultrasound images. 
+Given a  CNN trained on a  2D ultrasound dataset acquired using a transducer of a particular frequency, we can simulate ultrasound images and 3D volumes corresponding to various transducer frequencies without the need for availability of additional training data in 3D and at a different frequency. We also demonstrate the ability to simulate ultrasound images of breast tumour using a model trained on IVUS dataset. We have performed an extensive validation of our approach using both real and simulated ultrasound images. Using regression analysis, we have concluded that there exists a linear relationship between the proposed kernel interpolation factor and the transducer frequency, thereby laying the foundation of a multi-frequency full 3D ultrasound volume simulator.
+
+
+## Proposed Approach
+
+<p align="center"><img src="./media/graphic.jpg" alt="drawing" width="500"/></p>
+
+## Results
+
+![](./media/cart.jpg)
+
+
+## Datasets
+The network is trained on [IVUS](http://www.cvc.uab.es/IVUSchallenge2011/dataset.html) dataset. Data augmentation is applied as explain in [1,2,3] and images are processed by [stage 0](https://in.mathworks.com/matlabcentral/fileexchange/34199-pseudo-b-mode-ultrasound-image-simulator). We have directly provided the processed dataset [here]([link](https://drive.google.com/drive/folders/1d4iu2OHxSaORK4mPXqb0Wy2ivXbXZIdI?usp=sharing)). 
+
+Segmentation masks from [INbreast](http://medicalresearch.inescporto.pt/breastresearch/index.php/Get_INbreast_Database) dataset is used to generate stage 0 simulation results for breast ultrasound.
+
+
+## **Specification and Performance**
+
+
+| Type | Simulation |
+| --- | --- |
+| Source Framework | Pytorch (converted to ONNX) |
+| FLOPs | 34,027 |
+| params | 33,640 |
+
+## **Input and Output**
```

Comment:
There should be separate sections, Input and Output, to describe expected format of model's input and output blobs